### DISCUSSION TITLE
API to scrape Prometheus metrics for PostgreSQL?

### DISCUSSION THREAD

**Discussion by crusty-dave** [diesel-rs/diesel#3189] [Q&A]
Does diesel or any other Rust crate provide this capability?

Rust 1.47.0
Diesel 1.4.latest

**Comment by weiznich** [comment]
Diesel itself does not provide this functionality. I'm not aware of a third party crate which provides something like that specifically tailored to diesel/postgres

**Comment by mohe2015** [comment]
Why not use https://github.com/prometheus-community/postgres_exporter

**Reply by crusty-dave** [reply]
I have seen that, I was asking specifically about Rust solutions.
